projects/storefrontlib/src/cms-components/configurator/commons/config-attribute-types/config-attribute-checkbox-list/config-attribute-checkbox-list.component.html

```diff
@@ -0,0 +1,20 @@
+<div id="{{ uiKeyGenerator.createAttributeIdForConfigurator(attribute) }}">
+  <div class="form-check" *ngFor="let value of attribute.values; let i = index">
+    <div class="cx-config-select-line">
+      <input
+        type="checkbox"
+        class="form-check-input"
+        [value]="value.valueCode"
+        (change)="onSelect()"
+        [formControl]="attributeCheckBoxForms[i]"
+        name="abc123"
+      />
+    </div>
+    <div class="cx-config-attribute-value-label-container">
+      <label
+        class="cx-config-attribute-value-label form-check-label form-radio-label"
+        >{{ value.valueDisplay }}</label
```

Comment:
id and for attributes are missing for label tag.
Furthermore aria-attributes are missing